### DISCUSSION TITLE
Database Size Check Throw error when i used PostgreSQL Database

### DISCUSSION THREAD

**Discussion by Zubairsoft** [spatie/laravel-health#159] [Q&A]
- Laravel v9.40
- Postgres v15.2

I am using this package to check my application health and everything is going well. However, when I checked the size of the database using the `DataBaseSizeCheck` class, it threw an error:

Spatie\Health\Support\DbConnectionInfo::getPostgresDatabaseSize(): Return value must be of type int, string returned

  at vendor/spatie/laravel-health/src/Support/DbConnectionInfo.php:67
     63▕     protected function getPostgresDatabaseSize(ConnectionInterface $connection): int
     64▕     {
     65▕         return $connection->selectOne('SELECT pg_size_pretty(pg_database_size(?)) AS size;', [
     66▕             $connection->getDatabaseName(),
  ➜  67▕         ])->size;
     68▕     }
     69▕ }
     70▕ 

I think this issue is related to the return type hint of `getPostgresDatabaseSize()` method it actually returns a string but the return type hint is an integer.

Thank you.

**Comment by Zubairsoft** [comment]
@freekmurze  @eduardocruz  @dominikkukacka  @tominon

**Reply by SUXUMI** [reply]
Yeap, I also faced the same`TypeError` - `Spatie\Health\Support\DbConnectionInfo::getPostgresDatabaseSize(): Return value must be of type int, string returned`. Hope they will fix this small issue asap.

https://github.com/spatie/laravel-health/blob/d1b4e93d3164eb87d6783a37885a5603edca66a6/src/Support/DbConnectionInfo.php#L63

**Comment by freekmurze** [comment]
We need the size as an `int` (or `float`) as we use that in the the db size check to determine if your db is too large.

I'm not a postgres user myself (this functionality was contributed). I'd appreciate a PR that fixes this, not by changing the typehint, but by making sure the function actual returns an int.

**Reply by SUXUMI** [reply]
Hi, @freekmurze so I've done a tiny update and maybe someone could check it https://github.com/spatie/laravel-health/pull/170. Thanks

**Reply by freekmurze** [reply]
Done, I've released a new version containing that fix. Thanks!

**Reply by SUXUMI** [reply]
Thank you very much
